server/src/main/java/org/elasticsearch/search/aggregations/metrics/InternalTopHits.java

```diff
@@ -191,7 +192,7 @@ public Object getProperty(List<String> path) {
 
     @Override
     public XContentBuilder doXContentBody(XContentBuilder builder, Params params) throws IOException {
-        searchHits.toXContent(builder, params);
+        ChunkedToXContent.wrapAsToXContent(searchHits).toXContent(builder, params);
```

Comment:
Question out of curiosity: I thought the final goal would be to convert everything in search to a "chunked" xcontent variant. This looks a bit like for aggregations we aren't? Is this reading correct, if so is this future work or does that mean aggregations remain larger "chunks"

Reply:
Yes, the plan is to do this bit-by-bit.  So we start off by chunking search into search hits and aggregations, and we can then look at breaking these chunks into smaller ones in the future.